[Bug]: Fix the typo in aarch64 AES MPULL macro. (was #8346)

### Contact Details

@fraggerfox

### Version

@master

### Description

Currently the build fails in FreeBSD aarch64 with the following message

`FreeBSD main-arm64-default-job-07 15.0-CURRENT FreeBSD 15.0-CURRENT 1500030 arm64`

```
wolfcrypt/src/cpuid.c:262:28: error: use of undeclared identifier 'CPUID_AARCH64_FEAT_PMULL'
  262 |             if (features & CPUID_AARCH64_FEAT_PMULL)
      |                            ^
1 error generated.
--- wolfcrypt/src/src_libwolfssl_la-sp_int.lo ---
--- wolfcrypt/src/src_libwolfssl_la-cpuid.lo ---
*** [wolfcrypt/src/src_libwolfssl_la-cpuid.lo] Error code 1

make[2]: stopped making "all-am" in /wrkdirs/usr/ports/security/wolfssl/work/wolfssl-5.7.6
```

### Reproduction steps

_No response_

### Relevant log output

_No response_

--- a/wolfcrypt/src/cpuid.c
+++ b/wolfcrypt/src/cpuid.c
@@ -259,8 +259,10 @@
 
             if (features & CPUID_AARCH64_FEAT_AES)
                 cpuid_flags |= CPUID_AES;
-            if (features & CPUID_AARCH64_FEAT_PMULL)
+            if (features & CPUID_AARCH64_FEAT_AES_PMULL) {
+                cpuid_flags |= CPUID_AES;
                 cpuid_flags |= CPUID_PMULL;
+            }
             if (features & CPUID_AARCH64_FEAT_SHA256)
                 cpuid_flags |= CPUID_SHA256;
             if (features & CPUID_AARCH64_FEAT_SHA256_512)

